test/javascripts/fixtures/site-fixtures.js.es6

```diff
@@ -19,7 +17,15 @@ export default {
         moved_post: 10,
         linked: 11,
         granted_badge: 12,
-        liked_consolidated: LIKED_CONSOLIDATED_TYPE
```

Comment:
Is there a reason for us to not import the constant so as to avoid duplication?

Reply:
I preferred to do it this way for consistency. For me, it's fine either way. In my opinion, it doesn't matter so much since it's a test fixture.

Reply:
Usually we don't want to hard code "magic" numbers in multiple places. If the number changes, we end up having to change it in multiple places and we might not even know which are the places that should change.